# Description: disable corner typing when disabling tap to clik
# Upstream: https://bugzilla.gnome.org/show_bug.cgi?id=607474
# Ubuntu: https://bugs.launchpad.net/bugs/509724
--- a/plugins/mouse/gsd-mouse-manager.c
+++ b/plugins/mouse/gsd-mouse-manager.c
@@ -606,6 +606,9 @@ set_tap_to_click (gboolean state, gboolean left_handed)
 
                         if (rc == Success && type == XA_INTEGER && format == 8 && nitems >= 7)
                         {
+                                /* Set MR mapping for corner tapping on the right side*/
+                                data[1] = (state) ? 3 : 0;
+
                                 /* Set RLM mapping for 1/2/3 fingers*/
                                 data[4] = (state) ? ((left_handed) ? 3 : 1) : 0;
                                 data[5] = (state) ? ((left_handed) ? 1 : 3) : 0;

